app/Post_Types/Diviner_Field/Types/Select_Field.php

```diff
@@ -12,8 +13,16 @@ class Select_Field extends FieldType {
 	const TITLE = 'Select Field';
 	const TYPE = 'select';
 
-	static public function render( $id, $field_label, $helper = '') {
+	static public function render( $post_id, $id, $field_label, $helper = '') {
+		$options = carbon_get_post_meta( $post_id, FieldPostMeta::FIELD_SELECT_OPTIONS);
+		if (count($options) <= 1) {
+			return '';
+		}
 		$field =  Field::make( static::TYPE, $id, $field_label );
+		$filtered_options = array_map( function ($a) {
```

Comment:
use `array_column` if we're going for PHP 5.5+ compat or `wp_list_pluck` for compat w/ anything

Reply:
done